Public/Src/Engine/Distribution.Grpc/Interfaces.proto

```diff
@@ -103,49 +89,45 @@ message EventMessage
 // Information about completed pips and events logged on worker
 message WorkerNotificationArgs
 {
-    SenderInfo Sender = 1;
-
     // Identifies the worker which executed the pips
-    uint32 WorkerId = 2;
+    uint32 WorkerId = 1;

```

Comment:
Since nobody depends on our protobuf yet, this is okay to do since it is an internal implementaiotn detail, but when this protocol is used for a running service that upgrades SxS this is considerd a bad practice and one can remove fields but should never 'renumber'.

Reply:
thanks for letting me know!